Odoo.xml

```diff
@@ -57,62 +57,62 @@
       <option name="Python" value="true" />
     </context>
   </template>
-  <template name="odoo_field_integer" value="new_field = fields.Integer(string=&quot;&quot;, required=False, )" description="Add new field of type Integer" toReformat="false" toShortenFQNames="true">
+  <template name="odoo_field_integer" value="new_field = fields.Integer(string='', required=False, )" description="Add new field of type Integer" toReformat="false" toShortenFQNames="true">
```

Comment:
Great work, @mstuttgart . 
Can you help me to understand why you replaced the double qoutes to be the single quote !!

Thank you :)